trelloapp.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import os
+from flask import Flask, render_template
+import json
+from dotenv import load_dotenv  #to invoke .env file
+load_dotenv()
+
+trelloapp = Flask(__name__)
+
+#Module2 project #
+#APP_KEY = os.environ.get('APP_KEY')  # take env variable from local system envor
+#APP_TOKEN=os.environ.get('APP_TOKEN')
+
+APP_KEY = os.getenv("APP_KEY")  # take .env from dotenv
```

Comment:
How these are wired up should be documented in the README

Reply:
Documentation included in README. Thanks!